src/pages/home/Home.page.tsx

```diff
@@ -49,6 +49,13 @@ const statData: StatData[] = [
     icon: HiOutlineMail,
     percentage: "30%",
   },
+  {
+    id: 3,
+    label: "Total de orçamentos",
+    score: 100,
+    icon: HiOutlineMail,
+    percentage: "30%",
+  },
 ];
 
```

Comment:
Tip - The current use of a static array for the statistics data might not be the best approach if you plan on expanding or dynamically updating the data in the future. Consider fetching this data from an API or external source to improve flexibility and scalability.